src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/pkg/Package.kt

```diff
@@ -63,7 +63,7 @@ class Package private constructor() {
 
         const val OAKPAL_OPEAR_RESOURCES_PATH = "package/$OAKPAL_OPEAR_PATH"
 
-        const val NODE_TYPES_EXPORT_PATH = "package/nodetypes.export.cnd"
+        const val NODE_TYPES_SYNC_PATH = "package/nodetypes.sync.cnd"
```

Comment:
Maybe update readme, it still says `nodetypes.export.cnd`.

Reply:
yep, and I have few other things to update...

thanks for reminder